projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.html

```diff
@@ -21,9 +21,9 @@
       <span class="go-accordion-panel__title-text">{{ heading }}</span>
     </div>
     
-    <ng-container *ngIf="headerContent">
+    <div class="go-accordion-panel__projection">
```

Comment:
Would it be best to avoid rendering an empty div when there is no `headerContent`? Or does that not matter?

Reply:
We actually want to do this. The `div` will collapse if there is no content inside to fit the content from the other flex items, but it will grow to fill the available space as much as it can.